webclient/src/agave.ts

```diff
@@ -1,19 +1,27 @@
 import { CommandBuffer, COMMANDS } from "./commandbuffer";
 
+export type JSONValue = string | number | boolean | { [x: string]: JSONValue } | Array<JSONValue>;
+
 export class AgaveClient {
   private binarysocket0: WebSocket;
   private cb: CommandBuffer;
-  private session_name: string;
+  private sessionName: string;
   private onOpen: () => void;
-  private onJson: (json: any) => void;
-  private onImage: (data: string) => void;
+  private onJson: (json: JSONValue) => void;
+  private onImage: (data: Blob) => void;
 
   constructor(
     url = "ws://localhost:1235/",
     rendermode = "pathtrace",
-    onOpen = () => {},
-    onJson = (json: any) => {},
-    onImage = (data: string) => {}
+    onOpen = () => {
+      0;
```

Comment:
Similar note as on a different PR, might be cleaner to have this be a return statement

Reply:
Agreed - I will fix these.